project/tests/helpForm.test.js

```diff
@@ -0,0 +1,34 @@
+const { test, expect } = require('@playwright/test');
+const FormSteps = require('../steps/formSteps');
+const testData = require('../testData/testData.json');
+const logger = require('../../logger');
+
+test('Help form', async ({ page }) => {
+    const formSteps = new FormSteps(page);
+    logger.info('Starting Help form test');
```

Comment:
Logs should be in Elements, Steps and Browser classes

BaseElement:
async click() {
        logger.info(`Click on the element '${this.name}'`);
        ....
    }
Steps:
async clickHereButton() {
        logger.info('UI: click map button');
        await locatorPage.clickMapButtonButton();
    }
It allows better checks errors

Logs:
UI: click map button
Click on the element "Map button"
error: .....